airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -506,25 +515,24 @@ def execute(self, context, session=None):
 
     @AwsBaseHook.retry(should_retry_eni)
     def _start_wait_check_task(self, context):
-
         if not self.arn:
             self._start_task(context)
 
+        if not self.wait_for_completion:
+            return
+
```

Comment:
Whatever logs users were getting for the short period of time without a wait_for_completion they will no longer get. So we're calling it a bug fix with no deprecation?

Reply:
I'd agree with Raph, I dont think this is a desired behavior but more an forgotten edge case. I would call it a bug fix

Reply:
Ack, I'll call that quorum then, let's call it a bug fix :+1: